content/source/docs/enterprise/install/automating-the-installer.html.md

```diff
@@ -180,9 +180,10 @@ If you have chosen `production` for the `installation_type`, `production_type` i
 
 - `production_type` — One of `external` or `disk`.
 
-If you have chosen `disk` for `production_type`, `disk_path` is required:
+If you have chosen `disk` for `production_type`, `disk_path` and `pg_password` are required:
 
 - `disk_path` — Path on instance to persistent storage.
+- `pg_password` — The password for the internal PostgreSQL access.
```

Comment:
I would add another sentence detailing that it is auto-generated if not provided and that the user should consider explicitly setting their own value or retrieving the auto-generated value prior to re-installation.

Reply:
Added, thank you!